packages/bun-types/bun.d.ts

```diff
@@ -1,9 +1,11 @@
-import { SignalConstants } from "os";
-
 interface VoidFunction {
   (): void;
 }
 
+declare namespace BunJS {
```

Comment:
Could we just have this be `Bun`?

Reply:
Good call! Thought this would cause a conflict with the `Bun` global var but it doesn't